setup/debian-based.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+#This script sets up the pre-requisites for setting up app-studio in redhat based machines.
+# mainly setting up pip3,yq, kubectl, kustomize
+# Firsly it checks which type of system is running this
+# Then it installs the dependencies one by one
+sudo apt-get update -y
+sudo apt-get -y install pip3
```

Comment:
why pip3?

Reply:
I know that on Macs, it's necessary to specify pip3 instead of pip because the default Python for the OS is an older version.  I'm not sure if it's necessary for Debian-based systems.  Is there a problem with using pip3, @Michkov?

Reply:
no problem is that we do not need python at all, so no pip.

Reply:
Oh. When did we get rid of the python dependency? It was needed for one of the pre-reqs when we initially wrote the setup scripts, a couple of months ago.  Now if I could only remember which one...

Reply:
there are two versions of `yq` -> pip one and go lang one, we are using the golang one.

Reply:
These scripts are now using the correct version of yq and not using snap or pip.